src/server/scripts/Commands/cs_misc.cpp

```diff
@@ -2391,7 +2391,7 @@ class misc_commandscript : public CommandScript
 
             if (target->isDead())
             {
-                target->ToCreature()->Respawn();
+                target->ToCreature()->Respawn(true);
```

Comment:
Must be forced now since a GM hitting `.respawn` isn't going to know about (or care about) conditions and linked respawns.